docs/common-tasks/using-xdebug-with-phpstorm.md

```diff
@@ -3,97 +3,173 @@
 
 !!! important "Make sure your environment is up-to-date"
     In case there might be fixes for any problems you might encounter, consider [updating rig](upgrading-rig.md) before 
     proceeding.
 
-    Once done, run `rig doctor` to confirm Outrigger is in a healthy. Check out [Troubleshooting](../faq/troubleshooting.md) 
+    Once done, run `rig doctor` to confirm Outrigger is healthy. Check out [Troubleshooting](../faq/troubleshooting.md) 
     or the [F.A.Q.](../faq/general.md) if anything comes up.
 
     If you haven't updated your Docker Images in awhile, doing so now is a good precautionary step that you have everything 
     you need. Check out the [Routine Image Maintenance](../common-tasks/routine-image-maintenance.md)
 
+## Conceptual Requirements
+
+This section details the end configuration needs for Xdebug and connection between the containers
+and your IDE.
+
+For both browser and command line debugging PHPStorm needs to be configured to listen for the
+Xdebug connection and map paths of files as they appear in the container to how they appear
+to your IDE.
+
+If you are familiar with normal Xdebug setup, there are common steps which may be omitted.
+Specifically, there is no need to trigger the start of the debugger through query parameters,
+POST parameters, cookies or environmental variables. Containers are set to autostart debugging
+so as long as your IDE is listening debugging should beging automatically. 
+
+### Browser Based Debugging
+
+For browser based debugging ensure that your browser is connecting directly to the Apache-PHP
+container. This is because the IP address of the connection is used to connect back to your
+IDE. Chaining requests through intermediate containers like Varnish will result in the connection
+attempt for debugging getting made to the incorrect address. 
+
 ## Setup Steps
 
-### 1. Activate Xdebug for your running Drupal site
+### PhpStorm Configuration
 
-In your docker command or your docker-compose.yml manifest, ensure the environment variable `PHP_XDEBUG="true"`. This 
-will load the PHP Xdebug extension with the default configuration.
+#### Adjust the Project's Debug Settings
+       
+Open your project's settings and go to Languages & Frameworks -> PHP -> Debug. You can get to
+the project settings by going to: PhpStorm > Preferences (OS X) or File > Settings (Windows,
+Linux).
 
-For details of the Xdebug configuration of Phase2's Apache PHP containers, check out the 
-[apache-php-base Docker Hub page](https://hub.docker.com/r/phase2/apache-php-base/).
+![Debug Project Settings](../readme_images/debug_preferences.png)
 
-### 2. Configure PhpStorm for Xdebug
+- Ignore external connections can be set as desired, though leaving it unselected can ease setup.
+- Set Break at first line as desired. Without this checked, you will need to set breakpoints in
+  your code.
+- Max simultaneous connections is recommended to be greater than 1 to facilitate handling multiple
```

Comment:
Slightly confusing sentence structure. Maybe:

> Set **Max simultaneous connections** greater than "1" to facilitate handling multiple requests at once.

We can be more concise by just presenting cause-and-effect statements that we think are valuable to consider.